### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/component/grammar.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#769] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/component/grammar.html

你可以在 Markdown 中通过 component 代码块快速添加组件。 配置 使用 你可以使用 component 代码块来在 Markdown 中添加组件。YAML 和 JSON 的数据格式均受支持: YAML JSON: 案例 这里的 <VPCard> 是已经全局注册的卡片组件。 上方的代码块和下方等价: 

https://theme-hope.vuejs.press/zh/guide/component/grammar.html

**Comment by WangQuan-cn** [comment]
how to control the number of cards per row? thx~
请问如何控制每行显示的卡片数量？

**Reply by Mister-Hope** [reply]
自己写一个外层 div 包一下，https://plugin-components.vuejs.press/zh/guide/content/card.html

![image](https://github.com/vuepress-theme-hope/giscus-discussions/assets/33315834/12f9b60e-8d7c-4744-bd11-411c33bbd113)
